tests/suites/0_stateless/20+_others/20_0001_planner_v2.sql

```diff
@@ -236,4 +236,8 @@ select '===Array Literal===';
 select [1, 2, 3];
 select [];
 
```

Comment:
How about adding a test whose tables have more than one column.
Such as 
```sql
mysql> select * from t1;
+------+------+
| a    | b    |
+------+------+
|    1 |    2 |
|    3 |    4 |
|    4 |    5 |
+------+------+
3 rows in set (0.03 sec)
Read 3 rows, 24.00 B in 0.005 sec., 624.54 rows/sec., 4.88 KiB/sec.

mysql> select * from t2;
+------+------+
| c    | d    |
+------+------+
|    2 |    5 |
|    3 |    6 |
|    1 |    7 |
+------+------+
3 rows in set (0.02 sec)
Read 3 rows, 24.00 B in 0.004 sec., 845.86 rows/sec., 6.61 KiB/sec.

select * from t2 where exists (select * from t1 where t1.a = t2.c);
ERROR 1105 (HY000): Code: 1065, displayText = Scalar subquery must return only one column.
(Now the error doesn't exist)
```

Reply:
It's not convenient to catch error in current test framework, thus we were always getting rid of the failure test.